apps/dexdrip/dexdrip.c

```diff
@@ -49,6 +49,8 @@ radio_channel: See description in radio_link.h.
 // if status_lights = 1; the yellow light flashes while actively scanning                           //
 // if a light is flashing for more than 10 minutes straight, it may not be picking up your dex      //
 //                                                                                                  //
+  static volatile BIT allow_alternate_usb_protocol = 0;
```

Comment:
Thank you for adding this in! I for one often have a test wixel powered through usb but hooked up through ble still so knowing this flexibility is great!